web/packages/teleterm/src/services/config/appConfigSchema.ts

```diff
@@ -182,6 +204,8 @@ const getDefaultKeymap = (
         openConnections: 'Ctrl+Shift+P',
         openClusters: 'Ctrl+Shift+E',
         openProfiles: 'Ctrl+Shift+I',
+        terminalCopy: 'Ctrl+Shift+C',
+        terminalPaste: 'Ctrl+Shift+V',
```

Comment:
How are people on Linux and Windows going to know that these are the shortcuts for copy and paste inside a terminal? We could include them in the context menu on right click, but I think that would only work on Linux, right? idk if there's a solution on Windows, but I guess people are used to using a mouse for that anyway.

Reply:
> We could include them in the context menu on right click, but I think that would only work on Linux, right? idk if there's a solution on Windows, but I guess people are used to using a mouse for that anyway.

Good news, we can add an accelerator (keyboard shortcut) to menu items. It will be shown on all platforms. 
The only problem I see, is that by default we set `terminal.rightClick` on Windows to `copyPaste`. Perhaps, we should revert it to the context menu? So users can learn these new shortcuts.

Reply:
The accelerator added to menu items is fantastic, wow.

Since the equivalent of `rightClick: 'copyPaste'` is the default on all the other three terminals on Windows, I think we should stick with it there. Windows users can learn the shortcuts from docs I suppose if they absolutely need to, assuming they won't just use right click.

Perhaps we could also modify the tab with keyboard shortcuts to show "Copy in terminal" and "Paste in terminal" if the platform is different than macOS? I think we don't need to show it on macOS since 99% of users are not going to ever change the default shortcut. It'd be mostly shown just for Windows and Linux users.

Another very cool thing we could add is "Show keyboard shortcuts" below "Open config file" in additional actions menu, but I think we could leave this for another time.

Reply:
>Since the equivalent of rightClick: 'copyPaste' is the default on all the other three terminals on Windows, I think we should stick with it there. Windows users can learn the shortcuts from docs I suppose if they absolutely need to, assuming they won't just use right click.

👍 

>Perhaps we could also modify the tab with keyboard shortcuts to show "Copy in terminal" and "Paste in terminal" if the platform is different than macOS?

This a really good idea, done.

>Another very cool thing we could add is "Show keyboard shortcuts" below "Open config file" in additional actions menu, but I think we could leave this for another time.

👍